lib/lte_link_control/lte_lc.c

```diff
@@ -1821,6 +1821,39 @@ int lte_lc_reduced_mobility_set(enum lte_lc_reduced_mobility_mode mode)
 	return 0;
 }
 
+int lte_lc_uiccpowersave_get(enum lte_lc_uiccpowersave_mode *mode)
+{
+	int ret;
+	uint16_t mode_tmp;
+
+	if (mode == NULL) {
+		return -EINVAL;
+	}
+
+	ret = nrf_modem_at_scanf("AT%UICCPOWERSAVE?", "%%UICCPOWERSAVE: %hu", &mode_tmp);
```

Comment:
I'm adding here a fundamental question. Do we need these functions for `AT%UICCPOWERSAVE` command at all? Or do you have some strategy which AT commands are wrapped into functions and which are not because we do not have functions for all AT commands.

I think the obvious AT commands are such that have complex syntax either in command, response, or notification. And those that have the notification in the first place. This `AT%UICCPOWERSAVE` doesn't belong into those categories but there may be other reasons to still do it.

Reply:
Even without LTE LC support, it's easy for an application to enable `%UICCPOWERSAVE`. I think pretty much the only benefit of having it in LTE LC is that it makes it more visible, the user doesn't have to read through the AT command specification to find it. I'm not sure this is good enough reason to have it in LTE LC, though. Especially, when we have two different ways to apply this simple configuration, a Kconfig option and a function call (actually two, because there's also a read function).

Reply:
> I think that it is one of those functionalities that has potentially immediate impact for most applications, hence it can be setup in the LTE link controller for all the applications/samples to benefit from.

Hmmm. mode 0 is the default behavior of the modem before this command was ever introduced. This command brings in additional modes (1-3) that does additional power savings in some use cases such as eDRX. So use of this AT command doesn't have much benefit for most applications. I'm not expecting this to be used much but there are certainly some customers who wants to look into it. And they might just as well use the AT command directly with their preferred mode.